src/atoms/prompt.jsx

```diff
@@ -140,60 +192,51 @@ class Prompt extends Component {
         };
 
         return (
-            <div className={containerClasses} id={id} style={style}>
-                {this.renderAction()}
+            <div
+                className={containerClasses}
+                id={id}
+                style={style}
+            >
+                {/* {this.renderAction()} */}
+                <div
+                    onClick={this.onClick}
+                    onKeyDown={() => {}}
```

Comment:
Should we have a `const noop = () => {};` outside of `render()` and use that instead?  I thought anonymous functions inside the `render()` was a no-no (essentially as bad as `.bind(this)` inside `render()`).